### PR TITLE
Suvela_Coding_exam3

### PR DESCRIPTION
**Pull request by SuvelaKoct2020**
exam3

### CODE REVIEW

File: src/suvela/Exam3.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			count = 0;
+			int no = arr[i];
+			for (int j = 0; j < arr.length; j++) {
+				if (no == arr[j]) {
+					count++;
+					arr[i] = 0;
+				}
+
+			}
+			if (count > 1 && arr[i] == 0)
```
Comment (MaulikTechnocredits) [inline src/suvela/Exam3.java:17]: why we need arr[i] == 0 condition ? I think only count condition would be sufficient.

File: src/suvela/Exam3.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+				System.out.println("Duplicate elements are:" + no);
+		}
+	}
+
+	void duplicateString(String[] s) {
+		for (int i = 0; i < s.length; i++) {
+			count = 0;
+			String s1 = s[i];
+			for (int j = 0; j < s.length; j++) {
+				if (s1 == s[j]) {
```
Comment (MaulikTechnocredits) [inline src/suvela/Exam3.java:27]: string should not be compared with == .
you should use .equals method

File: src/suvela/Exam3.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+					count++;
+					s[i] = null;
+				}
+			}
+			if (count > 1 && s[i] == null)
```
Comment (MaulikTechnocredits) [inline src/suvela/Exam3.java:32]: s[i] == null is not required.

**Review by MaulikTechnocredits — CHANGES_REQUESTED** [review]
PLease change code as suggested.

File: src/suvela/Exam3.java (hunk 1 above)
Comment (SuvelaKoct2020) [reply src/suvela/Exam3.java:17]: ok, got it.

File: src/suvela/Exam3.java (hunk 2 above)
Comment (SuvelaKoct2020) [reply src/suvela/Exam3.java:27]: changed.

File: src/suvela/Exam3.java (hunk 3 above)
Comment (SuvelaKoct2020) [reply src/suvela/Exam3.java:32]: got it.

**Pull request merged**